geo/Cargo.toml

```diff
@@ -31,7 +31,7 @@ proj = { version = "0.30.0", optional = true }
 robust = "1.1.0"
 rstar = "0.12.0"
 serde = { version = "1.0", optional = true, features = ["derive"] }
-i_overlay = { version = "2.0.0, < 2.1.0", default-features = false }
```

Comment:
Since i_overlay breaks API's on minor versions as a matter of policy, we need to keep i_overlay pinned to a minor version.